packages/realm/bindgen/src/templates/typescript.ts

```diff
@@ -210,13 +200,21 @@ export function generate({ spec: rawSpec, file }: TemplateContext): void {
     }
   `);
 
+  out("// Enums");
+  for (const e of spec.enums) {
+    // Using const enum to avoid having to emit JS backing these
+    out(`export enum ${e.jsName} {`);
```

Comment:
You lost the `const` in `const enum`

Reply:
This is on purpose. Having a `const enum` in a non .d.ts file made the tests fail as it didn't have values at runtime. I suppose TSC only inlines these if they're in a TS source file.